abn/core/service_test.go

```diff
@@ -0,0 +1,30 @@
+package core
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+)
+
+func TestService(t *testing.T) {
```

Comment:
This is a good test for `initializeServer` func. This is not quote a test for the Service itself... please see comment above on directly calling `Start`.

Reply:
modified along with structure of `Start()`